nc/models.py

```diff
@@ -60,7 +60,7 @@ class Stop(CachingMixin, models.Model):
     stop_id = models.PositiveIntegerField(primary_key=True)
     agency_description = models.CharField(max_length=100)
     agency = models.ForeignKey('Agency', null=True, related_name='stops')
-    date = models.DateTimeField()
+    date = models.DateTimeField(db_index=True)
```

Comment:
We have an unusual constraint/index handling process, since the data is refreshed by dropping/recreating the the DB during each import, so this index should also be added to [nc/data/copy.sql](https://github.com/OpenDataPolicingNC/Traffic-Stops/blob/dev/nc/data/copy.sql#L89-L98).

Reply:
Added!